mini_pupper_tracking/launch/tracking.launch.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+from launch import LaunchDescription
+from launch_ros.actions import Node
+
+def generate_launch_description():
+    return LaunchDescription([
+        # IMU Filter Madgwick Node
+        Node(
+            package='imu_filter_madgwick',
+            executable='imu_filter_madgwick_node',
+            name='imu_filter',
+            parameters=[{
+                'use_mag': False,  # Set to True if you have magnetometer data
+                'publish_tf': False,  # Set to True if you want TF frames
+                'world_frame': 'enu',  # Options: 'enu', 'ned', 'nwu'
+                'fixed_frame': 'odom',
+                'constant_dt': 0.0,     
+                'publish_debug_topics': False,
+                'gain': 0.1,    
+                'zeta': 0.0,    
+            }],
+            remappings=[
+                ('imu/data_raw', 'imu/data'),  # Input: read from existing /imu/data
```

Comment:
I usually get confused at this part, 
so for input you are converting incoming 'imu/data_raw' topic into 'imu/data' topic which imu_filter_madgwick_node can consume?

when output, imu_filter_madgwick_node outputs on the topic imu/data, and you remap to imu/qdata?

if possible, please select specific message names like /imu/filtered_with_madgwick because navigation packges usually use the very generic `/imu/data`